sdk/spring/compatibility-test.yml

```diff
@@ -0,0 +1,33 @@
+trigger: none
+pr: none
+
+variables:
+  - template: ../../eng/pipelines/templates/variables/globals.yml
+
+stages:
+  - stage:
+    jobs:
+      - job:
+
+        pool:
+          vmImage: "ubuntu-latest"
+
+        steps:
+          - script: |
+              pip install termcolor
+            displayName: 'python module install'
+          - script: |
+              python ./sdk/spring/scripts/compatibility_add_dependencymanagement.py -b 2.6.6 -c 2021.0.1
```

Comment:
This file does not exist, now it's just a place holder?

Reply:
`dependencymanagement` -> `dependency_management`

Reply:
Yes, it was just a template to create pipeline action, and it will work on this PR:https://github.com/Azure/azure-sdk-for-java/pull/28728